Makefile

```diff
@@ -161,8 +161,8 @@ go-test: go-test-unit
 # Unit tests. Local only.
 .PHONY: go-test-unit
 go-test-unit: vendor
-	go test -v ./...
-	
+	go test -v `go list ./... | grep -v /presctl`
```

Comment:
why don't you want to test or lint the cli?

Reply:
Lets take offline

Reply:
Lets take offline

Reply:
What's the decision?

Reply:
leave like this. as the Linting is actually done in the makefile of presctl (not the big one)
I will add in another PR a step to call this make file in the CI...